src/Examples/Advanced/CustomProperties/Dictionary_Txy_diagrams.ipynb

```diff
@@ -259,7 +269,11 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2021-03-05 10:08:54 [INFO] idaes.init.props: Property package initialization: optimal - Optimal Solution Found.\n",
+      "2021-05-14 15:33:16 [WARNING] idaes.core.util.scaling: Accessing missing scaling factor for props[1].mole_frac_phase_comp\n",
```

Comment:
Not particularly important, but it might be nice to get rid of these, or at least inform the user that they will see them and that they should see documentation/examples of the scaling tools to know more.